src/apps/backup/rubrik/restapi/mode/cache.pm

```diff
@@ -67,6 +73,10 @@ Create cache files (job mode could use it with --cache-use option).
 
 =over 8
 
+=item B<--filter-job-type>
+
+Filter jobs by job type.
```

Comment:
Maybe we can add the documentation link or the list of accepted string (same thing in the other file jobs.pm)